test/common/runtime/runtime_impl_test.cc

```diff
@@ -192,6 +192,12 @@ TEST_F(DiskLoaderImplTest, All) {
   // test_feature_false is not in runtime_features.cc and so is false by default.
   EXPECT_EQ(false, snapshot->runtimeFeatureEnabled("envoy.reloadable_features.test_feature_false"));
 
+  // Test that quic flags can be overriden.
+  EXPECT_EQ(true, snapshot->runtimeFeatureEnabled(
+                      "envoy.reloadable_features.quic_testonly_default_false"));
+  EXPECT_EQ(false, snapshot->runtimeFeatureEnabled(
+                       "envoy.reloadable_features.quic_testonly_default_true"));
```

Comment:
I think this test would fail without your PR.
Can we also test the underlying flag value?  Those checks will probably have to be define-guarded as well, or you can do them in the quiche flags test using TestScopedRuntime  and merging the value in

Reply:
done.